### PR TITLE
supress authentication details from request-response logging (b) fixes #34

### LINKED ISSUE #34: Supress Logging of Authorization Headers
It is ideal if the logging of request and response payloads does not include any details of the `Authorization` HTTP headers.  This change aims to avoid that data ending up logged when logging is enabled.

### PR DESCRIPTION
**Pull request by aplgithub**

### CODE REVIEW

**Comment by rvowles** [comment]
Can you create an issue and link to this PR please?

**Comment by aplgithub** [comment]
fixes #34

**Comment by aplgithub** [comment]
Sorry I can't seem to [link](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) it.  The UI elements don't seems to be active for me.  Can you see if are able to link it for me?

**Comment by rvowles** [comment]
its all linked :-) thanks!

File: libraries/servlet-spring-jersey/src/main/java/cd/connect/spring/jersey/JerseyApplicationBase.java [hunk 1]
```diff
@@ -2,31 +2,14 @@
...
 import org.slf4j.LoggerFactory;
 import org.springframework.context.ApplicationContext;
 
-import javax.ws.rs.core.Configurable;
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```
Comment (rvowles) [inline libraries/servlet-spring-jersey/src/main/java/cd/connect/spring/jersey/JerseyApplicationBase.java:10]: i set the standard on these as explicit imports for everything...

**Comment by rvowles** [comment]
If we could do explicit imports then I can merge and release for you.

**Comment by aplgithub** [comment]
Hi @rvowles ; that wildcard import should be removed now.

**Review by rvowles — APPROVED** [review]

**Pull request merged**